Adding some tips for tweets (#41)
nodejs/tweet: README.md

- We aim to support and amplify ecosystem members who are doing interesting and nice work.

This list should not be expected to be comprehensive nor exclusive, and may be clarified, ammended, or otherwise modified to ensure that the way the Social Team works in the context of our Twitter account is well represented here.

### Other tips for tweets:

> - If you are driving folks to a link, place that link at the end of the tweet so that Twitter displays the content the best way it knows how. (Twitter card or whatever it is called.)

Reviewer: ```suggestion
- If you are driving folks to a link, place that link at the end of the tweet so that Twitter displays the content as a Twitter Card.
```
